feature_engine/feature_builder.py

```diff
@@ -51,6 +51,11 @@ def __init__(
 
         # Reading chat level data (this is available in the input file path directly).
         self.chat_data = pd.read_csv(self.input_file_path, encoding='mac_roman')
+        self.input_columns = self.chat_data.columns
+        # Deriving the base conversation level dataframe.
+        # This is simply the unique rows across "batch_num", and "round_num".
+        # Assume that "batch_num", and "round_num" together form a primary key for this table.
+        self.conv_data = self.chat_data.groupby(["batch_num", "round_num"]).sum(numeric_only = True).reset_index().iloc[: , :2]
```

Comment:
Note: this needs to change to `conversation_num`.

Reply:
(Resolved!)

Reply:
Thank you for the quick fix!